pkgs/development/python-modules/pyobjc-core/default.nix

```diff
@@ -0,0 +1,116 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchFromGitHub,
+  fetchpatch,
+  darwin,
+  unittestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "pyobjc-core";
+  version = "10.3.1";
+  src = fetchFromGitHub {
+    owner = "ronaldoussoren";
+    repo = "pyobjc";
+    rev = "v${version}";
+    hash = "sha256-1IpOaXQpBUxpErC5KR5k7bru6mbxNQSaiJLk9iz5tpg=";
+  };
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+  build-system = [ setuptools ];
+
+  sourceRoot = "${src.name}/pyobjc-core";
+
+  patches = [
+    # Two patches from upstream that will be incorporated in the next release
+    ./patches/upstream-splitsig-nsurl.patch
+    (fetchpatch {
+      name = "002-splitsig.patch";
+      url = "https://github.com/ronaldoussoren/pyobjc/commit/190fe5368a431e518921b9fc2dc9d9f221333924.patch";
+      stripLen = 1;
+      sha256 = "sha256-lzhPwN8SnLYmFWofsgX5lG6Kh6288xFRvFlmgN+exCM=";
+    })
+    # Fix or skip problematic tests
+    ./patches/001-test_sockaddr.patch
```

Comment:
is it possible to replace these with disabledTests ? 

https://nixos.org/manual/nixpkgs/unstable/#using-pytestcheckhook

Reply:
i think i found answer

Reply:
Not with the unittestCheckHook, unfortunately. This is *old* code, predating pytest.